openvpn/openssl/crypto/cipheraead.hpp

```diff
@@ -0,0 +1,347 @@
+//    OpenVPN -- An application to securely tunnel IP networks
```

Comment:
This whole file uses a different code style than the ciphergcm.hpp it replaces (gnu style with tab=8) and also adds a lot of extra empty lines that were not there previously.

Reply:
> I think we prefer real names in commits. I am not sure if the CLA actually requires to have > real names in there or not.

Identity protection is a priority for AirVPN and we will never recommend our personnel, developers and anybody else working for AirVPN to reveal personal data in public or to entities that do not publish a clear personal data processing policy. The choice is personal and free. 

Forcing any developer or open source contributor to reveal personal data sounds like a terrible idea: various countries have prosecuted and persecuted developers of tools related to cryptography. 

Furthermore AirVPN operates even in countries controlled by human rights hostile regimes and supports NGOs and activists who could be targeted by those regimes, if a link is established. If identity disclosure and certification were mandatory for OpenVPN code contributions, we would be very surprised and we would invite you to consider the dangers and the privacy issues related to the requirement. 

On top of that, would you be able to guarantee that GDPR and any other data protection legal framework would be fully respected? How would you handle personal data? How would you process it and protect it?

That said, _promindsd_ is a certified and trusted AirVPN developer. All AirVPN data is available in AirVPN web site including owner name and company ID and address. We are confident that such information is sufficient.

Kind regards
AirVPN Staff

Reply:
I just said that we normally have real names on commits and that I am not lawyer and do not know how non real names work in a legal agreement like the CLA. Thus having a real name would have been an easy way out of this situation.

For non realnames a CLA signed the company AirVPN will probably also work.

Just do make a huge fuss over this, I am was just trying to find an easy way. to avoid a hard question. I am just trying to ensure that the CLA is met.